### DISCUSSION TITLE
I think there's a problem here

### DISCUSSION THREAD

**Discussion by zV3KT0R** [yanet-platform/yanet#201] [Q&A] [answered]
uint8_t physicalPort_flags[CONFIG_YADECAP_PORTS_SIZE];
but metadata->fromPortId can be greater than CONFIG_YADECAP_PORTS_SIZE

https://github.com/yanet-platform/yanet/blob/87748d7bf95f44ba21725197b0ef18e196310741/dataplane/worker.cpp#L1063

**Comment by TheRandomCharacter** [comment, marked as answer]
There should be conversion from dpdk port id that is stored in metadata to local to worker (contignuous and indexable)  port id;
(`base::permanently::PortMapper`)

created issue #203
